modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-10_{context}"]
+= Release notes for {pipelines-title} General Availability 1.10
+
+With this update, {pipelines-title} General Availability (GA) 1.10 is available on {product-title} 4.11 and later versions.
+
+[id="new-features-1-10_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.10.
+
+[id="pipelines-new-features-1-10_{context}"]
+=== Pipelines
+
+* With this update, you can specify environment variables in a `PipelineRun` or `TaskRun` pod template to override or append the variables that are configured in a task or step. Also, you can specify environment variables in a default pod template to use those variables globally for all `PipelineRuns` and `TaskRuns`. This update also adds a new default configuration `forbidden-ends` to filter the environment variables while propagating from pod templates.
+* With this update, for a cluster resolver, bundle resolver, and Git resolver, a pipeline sets the `ConfigSource` value to its source from where the remote resources were received. The `ConfigSource` parameter includes a source URL and hex-encoded sha256 checksum of the cluster resource.
+* This update adds a new boolean flag `enable-provenance-in-status` into the `feature-flags` config map to enable the provenance field in status to be populated. The provenance field in status records authenticated metadata about how a software artifact was built.
```

Comment:
This part is a bit difficult to read: "The provenance field in status records authenticated metadata (...).
Can you simply say "The field records authenticated metadata (...)." instead?